test/JDBC/upgrade/14_7/schedule

```diff
@@ -364,6 +364,7 @@ app_name
 case_insensitive_collation
 sys-has_perms_by_name
 sys-has_perms_by_name-dep
+bbf_view_def-before-14_7-or-15_2
```

Comment:
This should be changed after merging the 2_X_DEV PR.

Reply:
Done